asllib/Interpreter.ml

```diff
@@ -846,28 +810,31 @@ module Make (B : Backend.S) (C : Config) = struct
   (* Evaluation of Local Declarations *)
   (* -------------------------------- *)
   and eval_local_decl s ldi env m_init_opt : env maybe_exception m =
+    let () =
+      if false then Format.eprintf "Evaluating %a.@." PP.pp_local_decl_item ldi
+    in
     match (ldi, m_init_opt) with
     (* Begin LDDiscard *)
-    | LDI_Discard _ty, _ -> return_normal env |: SemanticsRule.LDDiscard
+    | LDI_Discard, _ -> return_normal env |: SemanticsRule.LDDiscard
```

Comment:
nit: can you please add one space between LDI_Discard and the following comma, please?